Somesh/HTML/Assignment6/que1Assign6.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Question1Assignment6</title>
+        <style>
+            h1,
```

Comment:
h1 And h2 should be on same line

Reply:
its not necessary and it was done by prettier